cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -73,8 +78,11 @@ def import_handler(request, course_key_string):
             # Do everything in a try-except block to make sure everything is properly cleaned up.
             try:
                 data_root = path(settings.GITHUB_REPO_ROOT)
-                course_subdir = "{0}-{1}-{2}".format(course_key.org, course_key.course, course_key.run)
-                course_dir = data_root / course_subdir
+                if library:
+                    courselike_subdir = "library-v1_{0}-{1}".format(course_key.org, course_key.run)
```

Comment:
I still question why "v1" is hardcoded here. If the library XML is being versioned, get the version from a constant in the library xmodule to make sure it stays in sync. If it is not being versioned, remove the "v1".